### PR TITLE
User metrics - CSV Downloads

### PR DESCRIPTION
**Pull request by Davidjwkinghorn**

### CODE REVIEW

File: src/api/handlers/deviceprofile.py [hunk 1]
```diff
@@ -95,17 +96,18 @@ def platform_metrics(self, request):
...
+        metric, err = self.service.metric_anonymous_community_users(community_id)
 
-    elif metric == "user_profiles":
-      metric, err = self.service.metric_user_profiles(context, args)
+      elif community_id and metric == "community_profiles":
+        metric, err = self.service.metric_community_profiles(community_id)
 
-    elif metric == "community_profiles":
-      metric, err = self.service.metric_community_profiles(context, args, community_id)
+      if community_id and metric is "community_profiles_over_time":
```
Comment (BradHN1) [inline src/api/handlers/deviceprofile.py:109]: This should be ==

File: src/api/handlers/deviceprofile.py [hunk 2]
```diff
@@ -95,17 +96,18 @@ def platform_metrics(self, request):
     metric = args.get("metric", None)
     community_id = args.get("community_id", None)
     
-    if metric == "anonymous_users":
-      metric, err = self.service.metric_anonymous_users(context, args)
+    if metric:
```
Comment (BradHN1) [inline src/api/handlers/deviceprofile.py:99]: Question - if metric is passed in, but not one of the expected choices, I think that should be an error.  Pass back InvalidResourceError perhaps?

Comment (Davidjwkinghorn) [reply src/api/handlers/deviceprofile.py:100]: Sounds good

**Review by BradHN1 — APPROVED** [review]
LGTM.  Want to get the visit_log change deployed
I'd like you to write unit tests for the new routes.

**Pull request merged**